scss/components/_accordion-menu.scss

```diff
@@ -51,24 +51,36 @@ $accordionmenu-submenu-toggle-height: $accordionmenu-submenu-toggle-width !defau
 $accordionmenu-arrow-size: 6px !default;
 
 @mixin zf-accordion-menu-left-right-arrows {
-  .is-accordion-submenu-parent:not(.has-submenu-toggle) > a {
-    position: relative;
-
-    &::after {
-      @include css-triangle($accordionmenu-arrow-size, $accordionmenu-arrow-color, down);
-      position: absolute;
-      top: 50%;
-      margin-top: -1 * ($accordionmenu-arrow-size / 2);
-      #{$global-right}: 1rem;
+  .is-accordion-submenu-parent {
+    &:not(.has-submenu-toggle) > a {
```

Comment:
This may be needed because of the `force-element-nesting` rule. I find this rule too restrictive and think that we should allow complex selectors when they cannot be marged.

See https://github.com/sasstools/sass-lint/issues/950

But `sass-lint` seems badly maintained now. 😞